crates/sui-types/src/error.rs

```diff
@@ -87,6 +87,8 @@ pub enum SuiError {
     DeleteObjectOwnedObject,
     #[error("Invalid Batch Transaction: {}", error)]
     InvalidBatchTransaction { error: String },
+    #[error("Size limit exceeded: {limit} is {value}")]
+    SizeLimitExceeded { limit: String, value: String },
```

Comment:
is `value` ever a non integer? mostly curious here

Reply:
Not at the moment, but it could be various different integer types so I just went with string here for simplicity. A Rust pro could probably tell me a better way to do this? :)

Reply:
I felt a size/count is a usize pretty much but I am fine with whatever

Reply:
I think usize is defined as the size of a pointer, so not necessarily to do with any generic concept of size/count

Reply:
well I meant as returned by `len()` ;) 
would tend to agree on your statement though

Reply:
`u64` a better reference I guess